packages/office-ui-fabric-react/src/components/Check/Check.scss

```diff
@@ -70,4 +71,28 @@ $checkBoxHeight: 16px;
     border-radius: 10px;
     border: 1px solid WindowText;
   }
+}
+
+:global(.ms-DetailsList--Original) {
+  .root {
+    width: $origCheckBoxHeight - 2px;
```

Comment:
What is the 2px value representing here? Maybe we could make it a sass variable.